roadmap.md

```diff
@@ -0,0 +1,87 @@
+# Nix is everywhere and invisible
```

Comment:
The file (and PR) call this a roadmap, but it does not outline a medium term planning.
May I suggest "mission statement" instead, for the file name and for the title:

```suggestion
# Detailed Mission Statement
```

I'm not corporate enough to come up with a better term ;), but I think this would be an improvement.

Reply:
+1. "mission statement", "vision statement", "project objectives" / "project goals", all sound good to me.

Reply:
Yes, I ninjedited the PR description to call it a vision.

```suggestion
# Our Vision for Nix
```

Reply:
Maybe `roadmap.md` -> `vision.md` once the other items are taken care of too (so as not to loose line comments)?

Reply:
> invisible

Is that the right word? Nix can't really be "invisible" since people actually interact with it

Reply:
some ideas
 - and works transparently (- as possible?)
 - and gets out of the way
 - and is a robust utility (in the sense of electricity etc)
 - and is an expected but boring part of modern computing

Reply:
I really like "utility" I will leave some comments on that.